crates/turbopack-tests/tests/snapshot/basic-tree-shake/.require-side-effect/input/index.js

```diff
@@ -0,0 +1 @@
+const {cat} = require('./lib')
```

Comment:
This test case is postponed. I'll work on cjs analyzer with another PR. (So I can start working on scope hoisting)